node_modules/ioslib/lib/simulator.js

```diff
@@ -95,11 +95,11 @@ const devicePairCompatiblity = exports.devicePairCompatiblity = {
 	},
 	'8.x': {             // Xcode 8.x
 		'9.x': {         // iOS 9.x
-			'2.x': true, // watchOS 2.x
+			'2.2': true, // watchOS 2.2
```

Comment:
I was unsuccessful in pairing a watchOS 2.0 or 2.1 simulator with any iOS simulator with Xcode 8 Beta 6. Furthermore, there is no watchOS 2.2.2 simulator, only 2.2 and since 2.2 is the last 2.x watchOS simulator runtime, that means we can safely say `2.2`.